Validation/root/TValCompare.cc

```diff
@@ -249,7 +249,7 @@ void TValCompare::Report(Option_t* Opt) {
 //_____________________________________________________________________________
 void TValCompare::Summary(Option_t* Opt) {
 
-  int n0=0,ns=0,n1=0,n2=0,n3=0,n10=0,n11=0,n100=0;
+  int n0=0,ns=0,ne=0,n1=0,n2=0,n3=0,n11=0,n100=0;
   TIter it(&fList);
```

Comment:
Thanks for introducing the enum.  I suggest following-through with n0 -> nPerfect, etc.